### PR TITLE
Alarm Panel card: number buttons outline and adjusted padding and font-size fixing #2812

### LINKED ISSUE #2812: Accessibility in new Alarm Panel
HA Version: 0.88.0
Frontend Version: 20190220.0

Not a frontend developer here, the following is just an opinion. 
The latest ui update of the alarm panel is less accessible on the default theme than the previous one. 
The UI is cool and I enjoy it, but when you put it in the perspective of someone who has an hard time reading (eyes, ...), some problems may arise.

I came up with this solution:

Change the font of the numpad to something bigger and add an outline to the button.

What do you think?

### PR DESCRIPTION
**Pull request by mr-awk**
Thanks to @bramkragten for helping out.

Outline on buttons and less padding.
Found a solution for the font-size issue before mwc-button height bug is fixed.

fixes #2812

### CODE REVIEW

**Comment by mr-awk** [comment]
image

**Comment by mr-awk** [comment]
it's not absolutely best but a possible solution before mwc-button height bug is sorted out.
image

File: src/panels/lovelace/cards/hui-alarm-panel-card.ts [hunk 1]
```diff
@@ -136,7 +136,7 @@ class HuiAlarmPanelCard extends LitElement implements LovelaceCard {
                 @click="${this._handleActionClick}"
                 outlined
               >
-                ${this._label(state)}
+                <h1 style="font-size:16px;">${this._label(state)}</h1>
```
Comment (bramkragten) [inline src/panels/lovelace/cards/hui-alarm-panel-card.ts]: Style should not be inline

File: src/panels/lovelace/cards/hui-alarm-panel-card.ts [hunk 2]
```diff
@@ -164,11 +164,14 @@ class HuiAlarmPanelCard extends LitElement implements LovelaceCard {
...
                           .value="${value}"
                           @click="${this._handlePadClick}"
-                          dense
+                          outlined
+                          raised
                         >
-                          ${value === "clear"
-                            ? this._label("clear_code")
-                            : value}
+                          <h1 style="font-size:16px;">
```
Comment (bramkragten) [inline src/panels/lovelace/cards/hui-alarm-panel-card.ts]: Don't use `h1` inside a button, that's a header tag

**Comment by bramkragten** [comment]
I don't thing this looks good, let's keep all the font sizes normal, and just add a outline on the buttons.

**Comment by mr-awk** [comment]
> I don't thing this looks good, let's keep all the font sizes normal, and just add a outline on the buttons.

Okey, thought this could be a possible solution. I know it is not a good looking or very smart but it works, and help for people who is visual impaired.

**Comment by bramkragten** [comment]
I suggest this:
image

I can push my changes to your branch if you want?
Removing the base unit will make sure that the font sizes can scale when settings change. Not sure if it is possible to set font-size with a theme, but we might want to look into that.

**Comment by mr-awk** [comment]
No problem, I can fix it. Just missed the raised part. Will push that change in one-two hours.

Best regards,
Marcus

**Comment by mr-awk** [comment]
> I suggest this:
> image
> 
> I can push my changes to your branch if you want?
> Removing the base unit will make sure that the font sizes can scale when settings change. Not sure if it is possible to set font-size with a theme, but we might want to look into that.
> 
> With a base font of 20px instead of 14px it would look like this:
> image

@bramkragten Sorry didn't read your whole post. Please push your changes!

**Comment by iantrich** [comment]
ah, nice find @bramkragten. That makes sense now.

**Pull request merged**